borgcron.sh

```diff
@@ -216,15 +228,15 @@ for i in $( seq "$REPEAT_NUM" ); do
 			info_log "Borg has been successful."
 			;;
 		* )
-			error_log "Unknown error with code ${errorcode} happened."
+			error_log "Unknown error with code "$EXITCODE" happened."
 			;;
 	esac
 
 	# optional user question
 	guiTryAgain || break;
 
 	# exit on non-critical errors (ignore 1 = warnings)
-	if [ ${errorcode} -le 1 ]; then
+	if [ "$EXITCODE_BORGBACKUP" -le 1 ]; then
```

Comment:
Note sure whether the quotes cause trouble here, but in the end it is a number, so quotes are not needed.